### PR TITLE
Update wrap for Kraken-Engine-0.0.7

### PR DESCRIPTION
**Pull request by durkisneer1**

### CODE REVIEW

File: subprojects/kraken-engine.wrap [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-source_url = https://github.com/durkisneer1/Kraken-Engine/releases/download/v0.0.6/Kraken-Engine-0.0.6.tar.gz
-source_filename = Kraken-Engine-0.0.6.tar.gz
-source_hash = 620327cb4f5bb1aec3c079509e62a80d29b373265092f714e1a459021ae0b46d
+directory = Kraken-Engine-0.0.7
+source_url = https://github.com/durkisneer1/Kraken-Engine/releases/download/v0.0.7/Kraken-Engine-0.0.7.tar.gz
+source_filename = Kraken-Engine-0.0.7.tar.gz
+source_hash = ef8e4577ac858646384b5fa36411f29bbb1135c6e4853b43a2f60028d4b7995c
 
 [provide]
 kraken-engine = kraken_engine_dep
```
Comment (neheb) [inline subprojects/kraken-engine.wrap:8]: No newline. hrm.

Comment (durkisneer1) [reply subprojects/kraken-engine.wrap:8]: vscode autoformat 💔 next time

Comment (neheb) [reply subprojects/kraken-engine.wrap:8]: from https://github.com/mesonbuild/wrapdb/pull/1806 : please install EditorConfig.

Comment (durkisneer1) [reply subprojects/kraken-engine.wrap:8]: Oh it wasn't just a nitpicky joke... alright I'll edit the file

**Pull request merged**